dataretrieval/nwis.py

```diff
@@ -714,7 +719,6 @@ def _read_json(json, multi_index=False):
                 merged_df = update_merge(merged_df, record_df, na_only=True,
                                          on=['site_no', 'datetime'])
 
-    merged_df = format_response(merged_df)
```

Comment:
Was this intentional to remove?

Reply:
Yes: In the two functions were _read_json is called format_response is called again on the output so it is not needed here. If we want to keep it we can pass kwargs to _read_json so that it can turn on/off multi_index.

Reply:
I see.